BlazarService/src/main/java/com/hubspot/blazar/zookeeper/QueueProcessor.java

```diff
@@ -49,12 +49,12 @@ public QueueProcessor(@Named("QueueProcessor") ScheduledExecutorService executor
                         ZooKeeperEventBus eventBus,
                         ObjectMapper mapper,
                         MetricRegistry metricRegistry,
+                        CachingMetricsService cachingMetricsService,
```

Comment:
Doesn't seem that CachingMetricsService and MetricRegistry are used in this class